modules/csharp/examples/code_snippets/Program.cs

```diff
@@ -968,27 +968,66 @@ private static void StartReplication()
         private static void VerboseReplicatorLogging()
         {
             // tag::replication-logging[]
+            // deprecated
             Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
             Database.SetLogLevel(LogDomain.Network, LogLevel.Verbose);
             // end::replication-logging[]
         }
 
+        private static void ConsoleLogging()
+        {
+            // tag::console-logging[]
+            Database.Log.Console.Domains = LogDomain.All; // <.>
+            Database.Log.Console.LogLevel = LogLevel.Verbose; // <.>
+
+            // end::console-logging[]
+        }
+
         private static void FileLogging()
         {
             // tag::file-logging[]
             var tempFolder = Path.Combine(Service.GetInstance<IDefaultDirectoryResolver>().DefaultDirectory(), "cbllog");
-            Database.Log.File.Config = new LogFileConfiguration(tempFolder);
-            Database.Log.File.Level = LogLevel.Info;
+            var config = new LogFileConfiguration(tempFolder) // <.>
+            {
+                MaxRotateCount = 5, // <.>
+                MaxSize = 10240, // <.>
+                UsePlainText = false  // <.>
+            };
+            Database.Log.File.Config = config; // Apply configuration
+            Database.Log.File.Level = LogLevel.Info; // <.>
+
             // end::file-logging[]
         }
 
         private static void EnableCustomLogging()
         {
             // tag::set-custom-logging[]
-            Database.Log.Custom = new LogTestLogger();
+            Database.Log.Custom = new LogTestLogger(LogLevel.Warning); // <.>
```

Comment:
```suggestion
            Database.Log.Custom = new LogTestLogger(); // <.>
            or
            var customLogger = new LogTestLogger { Level = LogLevel.Warning };
```
Base on the custom LogTestLogger class, there is no constructor taking any parameter, new LogTestLogger() is the correct way to init the LogTestLogger object. Or new LogTestLogger { Level = LogLevel.Warning }; to create LogTestLogger  with it's Level property set to LogLevel.Warning.

Reply:
Thank-you @Sandychuang8